### PR TITLE
address Eliza's review comments

### PR DESCRIPTION
**Pull request by sunshowers**

### CODE REVIEW

File: crates/newtype-uuid/src/lib.rs [hunk 1]
```diff
@@ -277,8 +284,7 @@ impl TypedUuidTag {
     ///
     /// # Panics
     ///
-    /// Panics if the above conditions aren't met. (This is a const fn, so it can't return an
-    /// error.)
+    /// Panics if the above conditions aren't met. Use [`Self::try_new`] to handle errors instead.
     pub const fn new(tag: &'static str) -> Self {
```
Comment (hawkw) [inline crates/newtype-uuid/src/lib.rs:288]: this should probably also have a `#[must_use]` on it
```suggestion
    #[must_use]
    pub const fn new(tag: &'static str) -> Self {
```

File: crates/newtype-uuid/src/lib.rs [hunk 2]
```diff
@@ -408,16 +416,19 @@ impl std::error::Error for ParseError {
...
 pub trait GenericUuid {
     /// Creates a new instance of `Self` from an untyped [`Uuid`].
-    fn from_untyped_uuid(uuid: Uuid) -> Self;
+    fn from_untyped_uuid(uuid: Uuid) -> Self
+    where
+        Self: Sized;
 
     /// Converts `self` into an untyped [`Uuid`].
-    fn to_untyped_uuid(self) -> Uuid;
+    fn into_untyped_uuid(self) -> Uuid
```
Comment (hawkw) [inline crates/newtype-uuid/src/lib.rs:424]: while we're `#[must_use]`ing things, i must use these, as well

**Review by hawkw — CHANGES_REQUESTED** [review]

File: crates/newtype-uuid/src/lib.rs (hunk 1 above)
Comment (sunshowers) [reply crates/newtype-uuid/src/lib.rs:288]: Ahh yes, thanks!

File: crates/newtype-uuid/LICENSE-APACHE [hunk 3]
```diff
@@ -0,0 +1 @@
+../../LICENSE-APACHE
```
Comment (hawkw) [inline crates/newtype-uuid/LICENSE-APACHE:1]: (asking because i'm curious) does `cargo publish` behave nicely with symlinks?

**Review by hawkw — APPROVED** [review]

File: crates/newtype-uuid/LICENSE-APACHE (hunk 3 above)
Comment (sunshowers) [reply crates/newtype-uuid/LICENSE-APACHE:1]: Yeah (a contributor added this to nextest and I had the same q then -- works great)

**Pull request merged**